Makefile

```
@@ -92,7 +92,7 @@ else
 		--env GITHUB_WORKSPACE="$(shell git rev-parse --show-toplevel)" \
 		--env RUN_LOCAL=true \
 		--env VALIDATE_ALL_CODEBASE=false \
-		--env FILTER_REGEX_EXCLUDE='^$$PWD/test-data/' \
+		--env FILTER_REGEX_EXCLUDE="^$(CURDIR)/test-data/" \
```

I assume this will break if the user has some regex-matching characters in their path.
Probably okay, but maybe a bit brittle

---

Wait, shouldn't this just be:
```suggestion
		--env FILTER_REGEX_EXCLUDE="^$(shell git rev-parse --show-toplevel)/test-data/" \
```
for consistency?
With this in mind, is it worth setting the paths (L88 and L89 and L90, L92) to variables to avoid the duplicate calls?

---

> Wait, shouldn't this just be:
> 
> for consistency? With this in mind, is it worth setting the paths (L88 and L89 and L90, L92) to variables to avoid the duplicate calls?

Yeah, thanks for catching this!

> I assume this will break if the user has some regex-matching characters in their path.

It won't. But the `git rev-parse --show-toplevel` is what it was meant to be.

---

While I was addressing your suggestion, I found another issue.
Fixed in latest commit and it looks to be working as expected now including your worthwhile suggestion.
Please give it another look now. Thanks.

---

@MatthewJohn

---

I'll merge it now. Please let me know if you find anything else to improve. Thanks.